charts/rancher-vsphere-csi/Chart.yaml

```diff
@@ -6,16 +6,16 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.99-0'
```

Comment:
Update this to `catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'`, and add catalog.cattle.io/kube-version: '>= 1.19.0-0 <= 1.23.0-0'.